tests/backend-configuration-test.js

```diff
@@ -190,40 +184,41 @@ describeApplication('With valid backend configuration', () => {
             }]
           }, 500);
 
-          SettingsPage.save();
+          return SettingsPage.save();
         });
 
-        it.still('still shows the form action buttons', () => {
-          expect(SettingsPage.$actions).to.exist;
+        it('still shows the form action buttons', () => {
+          expect(SettingsPage.hasVisibleActions).to.be.true;
```

Comment:
This was previously `true` in another test outside of this describe, so we should assert that this one is `always` true, just in case it disappears milliseconds after saving changes.  ... but again with the flakiness of `always` currently, we probably need to skip it 😞

Reply:
`it.always.skip` added